incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,44 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
```

Comment:
can multiple nodes start at the same time? I have challenges with that, and readyprobe helps with that.